src/index.html

```diff
@@ -1,17 +1,130 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
Blank lines between parent and child tags are redundant, fix it for all such cases
```suggestion
<html lang="en">
  <head>
```